src/components/DeskList/DeskList.tsx

```diff
@@ -14,10 +15,10 @@ export interface DeskListProps {
 }
 
 export const DeskList = (props: DeskListProps) => {
-
   // console.log('DeskList is rendering...');
-
   const [desks, setDesks] = useState<Desk[]>([]);
+  const [dialogIsOpen, setDialogIsOpen] = useState(false);
+  const [selectedDesk, setSelectedDesk] = useState<string>('')
```

Comment:
From types perspective, presence or absence of the value here should be implemented as `string | null`. Null means nothing is specified, while string (including an empty string) means something is specified.